force-app/main/default/classes/DMLOperations.cls

```diff
@@ -48,8 +48,11 @@ public with sharing class DMLOperations {
      */
     public static void createAccount(String name, String industry) {
         // Create a new Account and set the fields inline
-
+        Account acc = new Account();
+        acc.Name = name;
+        acc.Industry = industry;
```

Comment:
Can you do this by putting the values right into the constructor of the Account (inline)?